index.js

```diff
@@ -0,0 +1,102 @@
+let countriesData = [];  
+const countriesGrid= document.querySelector('.countries-grid');
+const searchInput= document.querySelector('.search-input');
+const dropdownWrapper = document.querySelector('.dropdown-wrapper');
+const dropdownHeader = document.querySelector('.dropdown-header');
+
+
+fetch('./CountriesData.json')
+  .then((response) => response.json())
+  .then((countries) => {
+    countriesData = countries;
+    renderCountreis(countries);
+  })
+  .catch((error) => console.error('Error fetching countries:', error));
+
+  //----------------render the countries function----------------
+
```

Comment:
You don`t need this comment because the name of the function makes it super clear